test/functional/discovered_hosts_controller_test.rb

```diff
@@ -2,13 +2,14 @@
 
 class DiscoveredHostsControllerTest < ActionController::TestCase
   setup :initialize_host
-  
+
   setup do
     @request.env['HTTP_REFERER'] = '/discovery_rules'
     @facts = {
       "ipaddress" => "192.168.100.42",
       "macaddress" => "AA:BB:CC:DD:EE:FF",
       "discovery_bootif" => "AA:BB:CC:DD:EE:FF",
+      "bios_vendor" => "QEMU",
```

Comment:
nitpick: maybe use `facts = @facts.merge({"bios_vendor" => "QEMU"})` in the test instead on adding it here? because we don't want future tests to keep adding facts that only they use